src/50_Samples_%26_Templates/Product.html

```diff
@@ -516,7 +516,7 @@ <h2 class="m1">Apple</h2>
                   on="select:AMP.setState({
                     selectedColor: event.targetOption,
                     selectedSlide: 0,
-                    selectedSize: product[event.targetOption].sizes[selectedSize] ? selectedSize : product[event.targetOption].defaultSize
+                    selectedSize: product[event.targetOption].sizes[selectedSize] != 'unavailable' ? selectedSize : product[event.targetOption].defaultSize
```

Comment:
Is it possible to get rid of the `unavailable` state? What would happened if we'd include only the available sizes in the state?